### DISCUSSION TITLE
フォークチャレンジ(commit-mateのトップページのSTEP1-7)における作業ブランチについて

### DISCUSSION THREAD

**Discussion by wako-p** [commit-mate/commit-mate.net#30] [1. How to join the commit-mate] [answered]
作業ブランチは特に切らずmainで作業→プルリクエストで良いですか？

**Comment by monsat** [comment, marked as answer]
自分のアカウントにフォークしたリポジトリであれば、main ブランチでもOKです。
もちろん一般的には別ブランチを作成するので、そのブランチからプルリクエストを送ってもらってもOKです。

**Reply by monsat** [reply]
いまちょうど、説明文を修正していたところでした笑

**Reply by wako-p** [reply]
そうなんですね、ありがとうございます！
では、別ブランチを作成してそこからプルリクエスト送ってみたいと思います。
